mava/systems/tf/mad4pg/training.py

```diff
@@ -563,15 +563,13 @@ def _forward(self, inputs: reverb.ReplaySample) -> None:
 
                 # Critic loss.
                 critic_loss = recurrent_n_step_critic_loss(
-                    q_values,
-                    target_q_values,
-                    rewards[agent],
-                    discount * agent_discount,
+                    q_values=q_values,
+                    target_q_values=target_q_values,
+                    rewards=rewards[agent],
+                    discounts=discount * agent_discount,
                     bootstrap_n=self._bootstrap_n,
                     loss_fn=losses.categorical,
                 )
-                self.critic_losses[agent] = tf.reduce_mean(critic_loss, axis=0)
-
                 # Actor learning.
                 obs_agent_feed = target_obs_trans[agent]
                 # TODO (dries): Why is there an extra tuple?
```

Comment:
Not sure what is meant by this?

Reply:
It doesn't really matter. There is an extra [0] that is needed. But I removed the message as it does not really matter.